src/About/About.js

```diff
@@ -0,0 +1,224 @@
+                {/* <Typography className={classes.secondaryHeading}>I am an accordion</Typography> */}
+              </AccordionSummary>
+              <AccordionDetails>
+                <Typography>
+                  This is the core deepcell library.
+                  DeepCell-tf contains the necessary functions to train new deep learning models.
+                  The library has been constructed in a modular fashion to make it easy to mix and match different model architectures, prediction tasks, and post-processing functions.
+                  The <a href="https://github.com/vanvalenlab/deepcell-tf/blob/master/README.md">README</a> has instructions for getting started.
+                  For more information, check out the <a href="https://deepcell.readthedocs.io/en/master/">documentation</a>.
+                </Typography>
+              </AccordionDetails>
+            </Accordion>
+
+          </Grid>
+
+          <Grid item xs={10}>
+            <Typography variant="h4" className={classes.sectionHeader}>
+              Data
+            </Typography>
+
+            <Accordion>
+              <AccordionSummary
+                expandIcon={<ExpandMoreIcon />}
+                aria-controls="panel-data-1-content"
+                id="panel-data-1-header"
+              >
+                <Typography className={classes.heading}>caliban</Typography>
+                {/* <Typography className={classes.secondaryHeading}>I am an accordion</Typography> */}
+              </AccordionSummary>
+              <AccordionDetails>
+                <Typography>
+                  Caliban is our tool curating training data.
+                  It provides an inutitive GUI for users to create annotations from scratch, or to correct model predictions, to faciliate the creation of large, high-quality datasets.
+                  Caliban can be deployed locally, or to the cloud. The <a href="https://github.com/vanvalenlab/caliban/blob/master/README.md">README</a> has more instructions for how to get started.
+                </Typography>
+              </AccordionDetails>
+            </Accordion>
+
+          </Grid>
+
+          <Grid item xs={10}>
+            <Typography variant="h4" className={classes.sectionHeader}>
+              Deployment & Inference
+            </Typography>
+
+            <Accordion>
+              <AccordionSummary
+                expandIcon={<ExpandMoreIcon />}
+                aria-controls="panel-deployment-1-content"
+                id="panel-deployment-1-header"
+              >
+                <Typography className={classes.heading}>deepcell.org</Typography>
+                {/* <Typography className={classes.secondaryHeading}>I am an accordion</Typography> */}
+              </AccordionSummary>
+              <AccordionDetails>
+                <Typography component={'span'}>
+                  The deepcell.org websites provides a web interface to access trained deep learning models.
```

Comment:
Do we want to point to DeepCell.org as a place that people should depend on to get access to models? Or should it stay a prototype/proof-of-concept?